netlogo-gui/src/main/app/AbstractTabsPanel.scala

```diff
@@ -0,0 +1,175 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.app
+import java.awt.{ Component }
+import java.awt.event.{ MouseEvent }
+import javax.swing.{ JFrame, JTabbedPane, SwingConstants }
+import javax.swing.plaf.ComponentUI
+
+import org.nlogo.app.codetab.{ CodeTab, ExtendedCodeTab, ExternalFileManager, MainCodeTab }
+import org.nlogo.app.interfacetab.InterfaceTab
+import org.nlogo.window.{ GUIWorkspace }
+
+// AbstractTabsPanel contains functionality common to Tabs and CodeTabsPanel
```

Comment:
Please clarify the policy on initializing and dating comments
Are you saying when I create a new class,  I need to add my initials to every comment line that I add?
I thought the initials were for commenting places were you made changes to existing code that needed explanation.

If I don't change part of a file, but comment it to make what it does clearer, then should I add my initials?

I am happy to make the changes, but want to make sure I understand, so I get them all when I go through the file.

Reply:
Is there a workflow you use for making use of GitHub in doing updates - and then pulling it back to the local repo, or do you just make changes locally?

Reply:
I'm just going to sign all my comments etc. locally.

Reply:
Because code comments have a tendency to get stale, I think it's a good idea to sign and date all of them.  It's just easier when reading code to know how recently a comment was made.  If a comment is too small to have a signature and date, it might best be removed.

You can make updates locally and push new commits to the branch when you're ready.  That will automatically update the PR.